crates/eth-rpc/src/servers/net_rpc.rs

```diff
@@ -23,12 +22,10 @@ impl NetApiServer for NetRpc {
     }
 
     fn peer_count(&self) -> Result<PeerCount> {
-        // TODO: replace jsonrpsee::types::ErrorObject with Provider::Error once NetRpc uses Provider
-        Err(EthApiError::<jsonrpsee::types::ErrorObject>::MethodNotSupported("eth_peerCount".to_string()).into())
+        Ok(PEER_COUNT.clone())
```

Comment:
you can just do *PEER_COUNT.

Reply:
PeerCount does not implement Copy trait, so can't dereference it and has to use clone()

Reply:
@Eikix, I have no issues with what u are suggesting, it was just that I thought since we have constants for all these properties, then might be having one for this would also be consistent.

But, I don't see any issue with `PeerCount::Number(0)`, @aniketpr01 I think we can rollback to that, and if we need this property a second time, we can refactor back to a lazy_static!

Reply:
Done!